documentation/modules/operators/con-operator-topic-names.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-topic-operator.adoc
+
+[id='con-operator-topic-names-{context}']
+= Topic naming conventions
+
+A `KafkaTopic` resource includes a name for the topic and a label that identifies the name of the Kafka cluster it belongs to. 
+
+.Label identifying a Kafka cluster for topic handling
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaTopicApiVersion}
+kind: KafkaTopic
+metadata:
+  name: topic-name-1
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  topicName: topic-name-1  
+----
+
+The label provides the cluster name of the `Kafka` resource.
+If the label does not match the Kafka cluster, the Topic Operator cannot see the `KafkaTopic`, and the topic is not created.
+
+NOTE: The label is only used when deploying the Topic Operator with the `Kafka` resource.
+When deploying the standalone Topic Operator, you specify a connection to a Kafka cluster instead.
```

Comment:
This is wrong. Labels are not a mechanism for determining where the operator connects. They are a mechanism for determining which `KafkaTopics` it tries to consume. So it's still possible to use labels with the standalone operator.

Reply:
changed as follows:

> The label provides the cluster name of the `Kafka` resource.
> The Topic Operator uses the label as a mechanism for determining which `KafkaTopic` resources to manage.
> If the label does not match the Kafka cluster, the Topic Operator cannot see the `KafkaTopic`, and the topic is not created.